contrib/containers/ci/Dockerfile

```diff
@@ -116,8 +116,8 @@ RUN apt-get update && apt-get install $APT_ARGS \
     wine64 \
     xorriso
 
-ARG CPPCHECK_VERSION=2.10
-RUN curl -sL "https://github.com/danmar/cppcheck/archive/${CPPCHECK_VERSION}.tar.gz" | tar -xvzf - --directory /tmp/
+ARG CPPCHECK_VERSION=2.13.0
+RUN curl -sL "https://codeload.github.com/danmar/cppcheck/tar.gz/refs/tags/${CPPCHECK_VERSION}" | tar -xvzf - --directory /tmp/
```

Comment:
why changing url here?

Reply:
I have no issues with the old url, try
https://github.com/danmar/cppcheck/archive/2.11.tar.gz
https://github.com/danmar/cppcheck/archive/2.13.0.tar.gz

Reply:
why this url diff still here?